src/components/modal/modal.tsx

```diff
@@ -137,7 +139,7 @@ export const Modal: FC<ModalProps> = ({
             )}
             {!withoutFooter &&
               (CustomFooter ? (
-                <CustomFooter closeHandler={closeModal} />
+                <CustomFooter closeHandler={closeModal} {...customFooterProps} />
```

Comment:
Let's update to `createFooter` render prop. See the comment in service ui - https://github.com/reportportal/service-ui/pull/3964#discussion_r1718242684